cosmwasm/contracts/wattpeak-minter/src/state.rs

```diff
@@ -2,6 +2,8 @@ use cosmwasm_schema::cw_serde;
 use cosmwasm_std::{Addr, Coin, Decimal, Deps, StdError, StdResult};
 use cw_storage_plus::{Item, Map};
 
+use crate::msg::Location;
```

Comment:
The state should not depend on the msg. If anything it should be the other way around. Put the location struct here and depend on it in the msg

Reply:
Done!